### DISCUSSION TITLE
请问模型文件放在哪里？（May I ask where the model files are located? ）

### DISCUSSION THREAD

**Discussion by old-xno** [linzepore/xeno_rasa#3] [Q&A] [answered]
请问模型文件放在哪里？（May I ask where the model files are located? ）

**Comment by linzepore** [comment, marked as answer]
模型文件位于Release目录：[https://github.com/linzepore/xeno_rasa/releases](https://github.com/linzepore/xeno_rasa/releases) ，一般而言推荐使用最新发布，也请留意Release处的相关说明
